extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -101,6 +103,13 @@ private function getUserById( $userId ) {
 			throw new \InvalidArgumentException( self::NO_USER_MATCH_ERROR );
 		}
 
+		$user->load();
+
+		// Anonymous users can't have contributions
+		if ( $user->mName === false ) {
```

Comment:
That won't work, because `isAnon()` calls `isLoggedIn()` which return `$this->getId() != 0;`. Since we're providing the user id, `isAnon()` will always return `false`. Perhaps I should update the comment - I guess it's not so much that the user is anonymous, but that MW is unable to get information for that user id.